src/io/flutter/editor/PropertyEditorPanel.java

```diff
@@ -330,8 +330,7 @@ InspectorObjectGroupManager getGroupManager() {
   int getOffset() {
     if (outline == null) return -1;
     final VirtualFile file = activeFile.getValue();
-
-    assert (activeFile.getValue() != null);
+    if (activeFile == null) return -1;
```

Comment:
We've already dereferenced `activeFile` above; perhaps you meant `file` here? We should also replace the `activeFile.getValue()` reference below with `file`.

Reply:
doh. meant to wrote file instead of activeFile. fixed